lightning/src/ln/onion_route_tests.rs

```diff
@@ -270,9 +278,10 @@ fn test_fee_failures() {
 	claim_payment(&nodes[0], &[&nodes[1], &nodes[2]], payment_preimage_success);
 
 	let (_, payment_hash, payment_secret) = get_payment_preimage_hash!(nodes[2]);
+	let short_channel_id = channels[0].0.contents.short_channel_id;
 	run_onion_failure_test("fee_insufficient", 0, &nodes, &route, &payment_hash, &payment_secret, |msg| {
 		msg.amount_msat -= 1;
-	}, || {}, true, Some(UPDATE|12), Some(NetworkUpdate::ChannelClosed { short_channel_id: channels[0].0.contents.short_channel_id, is_permanent: true}));
```

Comment:
I see this was already here but why would `UPDATE:12` result in `NetworkUpdate::ChannelClosed` instead of `NetworkUpdate::ChannelUpdate`?

Reply:
Heh, I had to go look up the same cause it was confusing - its because the update we got back shouldn't have resulted in a payment failure. They're telling us the fee they charged was, in fact, enough for the payment, implying the previous-hop node malleated the payment before forwarding, taking funds they weren't supposed to.

Reply:
Ah, I see. @jkczyz think we could add a comment for this one since we're here?